.github/workflows/check_reference.yaml

```diff
@@ -0,0 +1,33 @@
+name: Check reference strings
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/check_reference_strings.yaml'
```

Comment:
Inconsistent with this file name. Recommend updating the filename to `check_reference_strings.yaml` to match.

Reply:
Good catch. I renamed the file because it felt too long, but forgot to update the yaml file.

Reply:
(I will also switch to `.yml`, since the other files in the repo are using that extension.)